plugins/prometheus.go

```diff
@@ -0,0 +1,55 @@
+/*
+Prometheus exporter.
+* https://prometheus.io/docs/instrumenting/exposition_formats/
+
+Exposes all counters and numerical values as gauges. Formatting follows,
+
+# TYPE cc_[service]_[metric] gauge
+cc_[service]_[metric] VALUE TS
+*/
+package plugins
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+
+	"github.com/slvwolf/ccentral/client"
+)
+
+// UnixTime - Provides unix epoch in seconds
+type UnixTime interface {
+	Unix() int64
+}
+
+// GeneratePrometheusPayload - Collect and create Prometheus payload
+func GeneratePrometheusPayload(cc client.CCServerReadApi, unixTime UnixTime) ([]byte, error) {
+	epoch := unixTime.Unix()
+	var buffer bytes.Buffer
+	serviceList, err := cc.GetServiceList()
+	if err != nil {
+		log.Printf("WARN Could not retrieve service list")
```

Comment:
How fatal is this error? If it affects the caller, you should return the error instead of just discarding it. However, if it's not important, then this is fine.

Reply:
Errors now return status 500 instead